tests/utils.go

```diff
@@ -5187,3 +5187,33 @@ func MountCloudInitFunc(devName string) func(*v1.VirtualMachineInstance) {
 		Expect(err).ToNot(HaveOccurred())
 	}
 }
+
+func CreateAndWaitForVmi(client kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
+	vmi, err := client.VirtualMachineInstance(vmi.Namespace).Create(vmi)
+	if err != nil {
+		return nil, err
+	}
+
+	vmi = WaitUntilVMIReady(vmi, libnet.WithIPv6(console.LoginToCirros))
+	return vmi, nil
+}
+
+func DeleteVmi(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) error {
+	if vmi != nil {
+		return virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &metav1.DeleteOptions{})
+	}
+
+	By("vmi is nil")
+	return nil
+}
+
+func WaitForVmiDeletion(vmi *v1.VirtualMachineInstance) {
+	if vmi != nil {
+		EventuallyWithOffset(1, ThisVMI(vmi), 120).Should(BeGone())
+	}
+}
```

Comment:
Same, wrapping the nil condition is not nice.

Reply:
Fixed.